system/jlib/jtrace.cpp

```diff
@@ -69,3 +73,190 @@ const char* LogTrace::queryLocalId() const
 {
     return localId.get();
 }
+
+void TraceManager::initTracer()
+{
+    //Handle HPCC specific tracing configuration here
+    //Target exporter? exporter connection info?
+    //Target processor(s)? batch/interactive? prob not
+    //Target propogator? http? grpc? binary? custom?
+    //HPCC component tracing switches?
+    Owned<const IPropertyTree> traceConfig;
+    try
```

Comment:
I'm not sure you need the try block.  If this throws an exception it suggests you are calling it at the wrong point.